### PR TITLE
Updating support information

### PR DESCRIPTION
**Pull request by choldgraf**
This updates our pilot documentation to point to the new `support/` repository.

### CODE REVIEW

File: admin/howto/support.md [hunk 1]
```diff
@@ -1,52 +1,31 @@
...
+(support:issues)=
+## File a support ticket
 
-(support:slack)=
-## Connect on Slack
+We handle most support requests via [GitHub issues in the `support/` repository](https://github.com/2i2c-org/support/issues).
+This is a **private repository** that is accessible to Community Champions and 2i2c Engineers.
 
-You're also welcome to connect with the 2i2c Hub Engineering team via [the 2i2c Slack](https://2i2c.slack.com) (in particular, check out the `#support` channel). If you don't have access to the Slack already, [open an issue in the pilot repository](https://github.com/2i2c-org/pilot/issues/new) and we're happy to give you access.
+[**Click here to file a support ticket**](https://github.com/2i2c-org/support/issues/new).
```
Comment (choldgraf) [inline admin/howto/support.md:16]:
```suggestion
[**Click here to file a support ticket**](https://github.com/2i2c-org/support/issues/new) or go to [support.2i2c.org](http://support.2i2c.org).
```

File: admin/howto/support.md [hunk 2]
```diff
@@ -1,52 +1,31 @@
...
 
-(support:slack)=
-## Connect on Slack
+We handle most support requests via [GitHub issues in the `support/` repository](https://github.com/2i2c-org/support/issues).
+This is a **private repository** that is accessible to Community Champions and 2i2c Engineers.
 
-You're also welcome to connect with the 2i2c Hub Engineering team via [the 2i2c Slack](https://2i2c.slack.com) (in particular, check out the `#support` channel). If you don't have access to the Slack already, [open an issue in the pilot repository](https://github.com/2i2c-org/pilot/issues/new) and we're happy to give you access.
+[**Click here to file a support ticket**](https://github.com/2i2c-org/support/issues/new) or go to [support.2i2c.org](http://support.2i2c.org).
 
+:::{note}
```
Comment (damianavila) [inline admin/howto/support.md:18]: You are saying the same again a little bit below... it feels a little bit repetitive.

File: admin/howto/support.md [hunk 3]
```diff
@@ -1,52 +1,31 @@
...
 
-For support requests that should not be public, you can also [send the 2i2c engineering team an email via `support@2i2c.org`](mailto:support@2i2c.org).
-We prefer to handle support requests via public issue tracking, but understand that this is not always possible!
+For support requests that have sensitive information, you can also [send the 2i2c support team an email via `support@2i2c.org`](mailto:support@2i2c.org).
+We prefer to handle support requests via the `support/` repository tracking, but understand that this is not always possible!
+
+(support:slack)=
+## Connect on Slack
+
+You're also welcome to connect with the 2i2c Hub Engineering team via [the 2i2c Slack](https://2i2c.slack.com) (in particular, check out the `#support` channel). If you don't have access to the Slack already, [open an issue in the support repository](https://github.com/2i2c-org/support/issues/new) and we're happy to give you access.
```
Comment (damianavila) [inline admin/howto/support.md:31]: Maybe we can word this in another way to discourage dropping issues on Slack... IMHO, the Slack support channel should be a place to talk with the community champion to gather **additional** information...

**Comment by yuvipanda** [comment]
I opened https://github.com/2i2c-org/team-compass/issues/151 to talk about issues wrt privacy and support.

**Comment by choldgraf** [comment]
Hey all - I'm closing this one because we decided to try FreshDesk in https://github.com/2i2c-org/team-compass/issues/151, and don't want to get this in the TC only to immediately switch.

**Pull request closed without merging**